{"CVEs":["CVE-2013-2132"],"threat":["An attacker could use this issue to cause PyMongo to crash, resulting in a denial of service.."],"date":"July 03, 2013","info":"pymongo vulnerability","usn":"USN-1897-1","update":{"13.04":[{"version":"2.2-4ubuntu0.1","package":"python-bson"},{"version":"2.2-4ubuntu0.1","package":"python-bson-ext"}],"12.10":[{"version":"2.2-2ubuntu0.1","package":"python-bson"},{"package":"python-bson-ext","version":"2.2-2ubuntu0.1"}],"12.04":[{"package":"python-bson","version":"2.1-1ubuntu0.1"}]},"special":"The problem can be corrected by updating your system to the following package versions:\nIn general, a standard system update will make all the necessary changes."}